lib/mp4/transmuxer.js

```diff
@@ -1145,4 +1164,6 @@ module.exports = {
   Transmuxer: Transmuxer,
   VideoSegmentStream: VideoSegmentStream,
   AudioSegmentStream: AudioSegmentStream,
+  audioProperties: audioProperties,
+  videoProperties: videoProperties
```

Comment:
I'm not 100% consistent on it myself but I try to do ALL_CAPS_CASE for constants.